src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -45,7 +45,10 @@ public NodeAutoCompleteSearchControl()
             if (Application.Current != null)
             {
                 Application.Current.Deactivated += CurrentApplicationDeactivated;
-                Application.Current.MainWindow.Closing += NodeAutoCompleteSearchControl_Unloaded;
+                if (Application.Current.MainWindow != null)
+                {
+                    Application.Current.MainWindow.Closing -= NodeAutoCompleteSearchControl_Unloaded;
```

Comment:
it's ok the -= ? or should it be +=? like the line deleted?

Reply:
nice catch